### PR TITLE
Implements a new layout for the login page

### PR DESCRIPTION
**Pull request by raphaabrasil**

### CODE REVIEW

**Comment by raphaabrasil** [comment]
This PR changes the login screens layout from this ones:
image
image

to this ones:
image
image

File: tsuru_dashboard/static/css/abyss.login.css [hunk 1]
```diff
@@ -3,34 +3,61 @@ body {
 	padding-bottom: 40px;
 	background-color: #222233;
 }
+
+.login-container {
+	top: 50%;
+  left: 50%;
```
Comment (andrewsmedina) [inline tsuru_dashboard/static/css/abyss.login.css]: can you fix the indentation in this file?

**Comment by andrewsmedina** [comment]
@raphaabrasil the logo it's too big to me

**Comment by raphaabrasil** [comment]
@andrewsmedina i was using 650px for the logo size. I've now changed to 550px and the result was:
OAuth screen:
image
Basic auth screen:
image

I would like to know what do you think about it before i push the changes.

**Comment by andrewsmedina** [comment]
@raphaabrasil LGTM

File: tsuru_dashboard/static/css/abyss.login.css [hunk 2]
```diff
@@ -43,15 +70,59 @@ body {
...
 	list-style: none;
-	padding: 5px;
 	font-size: 0.9em;
 }
 .form-actions {
 	margin: 0px;
 }
+.form-actions .button-login {
+	text-transform: uppercase;
+  color: #fff;
```
Comment (ggarnier) [inline tsuru_dashboard/static/css/abyss.login.css]: There are still a couple of lines with wrong indentation. Can you check this for the whole file?

Comment (raphaabrasil) [inline tsuru_dashboard/static/css/abyss.login.css]: Sure! i'll reindent the whole file just for guarantee.

Comment (raphaabrasil) [inline tsuru_dashboard/static/css/abyss.login.css]: @ggarnier done!

**Comment by ggarnier** [comment]
@raphaabrasil thank you!

**Pull request merged**